examples/lighting-app/esp32/sdkconfig.defaults.esp32h2

```diff
@@ -20,7 +20,7 @@ CONFIG_BT_ENABLED=y
 CONFIG_BT_NIMBLE_ENABLED=y
 CONFIG_BT_NIMBLE_EXT_ADV=n
 CONFIG_BT_NIMBLE_HCI_EVT_BUF_SIZE=70
-CONFIG_DEINIT_BLE_ON_COMMISSIONING_COMPLETE=n
+CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING=n
```

Comment:
Why is it set to `n`? I think it should be removed from here

Reply:
Not sure, but it was there before so did not change it, just renamed.

Reply:
@chshu  Can you please verify this. IMO, we can free the BLE memory after commissioning for H2 too in the default cases

Reply:
Please keep it for now (with this MR), we will have to double check the BLE deinit feature on H2.

Reply:
Talked with @chshu. He mentioned to keep this as is for now. Once we confirm about H2 then we can update it